src/django_components/dependencies.py

```diff
@@ -545,27 +551,27 @@ def on_replace_match(match: "re.Match[bytes]") -> bytes:
 
         # Schedule to load the `<script>` / `<link>` tags for the JS / CSS variables.
         # Skip if no variables are defined.
-        if js_input_hash is not None:
-            inputs_data.append((comp_cls_id, "js", js_input_hash))
-        if css_input_hash is not None:
-            inputs_data.append((comp_cls_id, "css", css_input_hash))
+        if js_variables_hash is not None:
+            variables_data.append((comp_cls_id, "js", js_variables_hash))
+        if css_variables_hash is not None:
+            variables_data.append((comp_cls_id, "css", css_variables_hash))
 
     (
-        to_load_input_js_urls,
-        to_load_input_css_urls,
-        inlined_input_js_tags,
-        inlined_input_css_tags,
-        loaded_input_js_urls,
-        loaded_input_css_urls,
-    ) = _prepare_tags_and_urls(inputs_data, type)
+        js_variables_urls_to_load,
```

Comment:
Unrelated to the issue, but I renamed these internal variables so it's clearer what they do.